cmake/shields.cmake

```diff
@@ -0,0 +1,19 @@
+if(CMAKE_SCRIPT_MODE_FILE AND NOT CMAKE_PARENT_LIST_FILE)
+  # This file was invoked as a script directly with -P:
+  # cmake -P shields.cmake
+  #
+  # Unlike boards.cmake, this takes no OUTPUT_FILE option, but
+  # SHIELD_LIST_SPACE_SEPARATED is required.
+  string(REPLACE " " ";" SHIELD_LIST "${SHIELD_LIST_SPACE_SEPARATED}")
```

Comment:
which means you can avoid turning the space separated list into a CMake list, as it is already a CMake list.
Just use the list, as a list. That's the pro-tip :wink: 

```suggestion
```

Reply:
>  Just use the list, as a list. That's the pro-tip wink

Escaping the semicolons was what I was missing. Thanks.